tests/casestmt/tcasestmt.nim

```diff
@@ -50,7 +50,7 @@ block emptyset_when:
     of {}: echo "me not fail"
     of 2: echo "i am always two"
     of []: echo "me neither"
-    of {4,5}: echo "set is 4 not 5"
+    of {range[4..5](4),5}: echo "set is 4 not 5"
```

Comment:
This makes no sense, sets in case statements are never materialized.

Reply:
We could also fix this by directly expanding `nkCurly` and `nkBracket` literals in case branches before evaluating them but it's not needed for now